### PR TITLE
[New] Added Coloured Initials to Results Cards

### PR DESCRIPTION
**Pull request by KateAThornley**
- Added avatarColor and nameInitials to getAllProfiles.js in backend
- Added avatars to results cards in ResultCardView

### CODE REVIEW

File: services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx [hunk 1]
```diff
@@ -10,20 +10,20 @@ function BasicInfoView(props) {
   /* Component Styles */
   const styles = {
     profileHeaderRow: {
-      margin: "25px 0",
```
Comment (dynamic11) [inline services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx]: I think we decided to leave these commas in. As trailing commas are recommended by some people to make diffs easier. We should discuss this with Risvi further.

**Review by dynamic11 — CHANGES_REQUESTED** [review]
We should discuss the comment as a team to come up with a solution so that it can become a standard.

File: services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx (hunk 1 above)
Comment (Rizvida1) [reply services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx]: I agree. Maybe your version of Prettier or VS Code is not up to date, Kat.

Comment (dynamic11) [reply services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx]: maybe we should add a prettier config file to resolve this issue in the future?

Comment (KateAThornley) [reply services/frontend-v3/src/components/basicInfo/BasicInfoView.jsx]: I fixed this- my prettier was out of date sorry guys!

**Review by dynamic11 — APPROVED** [review]

**Pull request merged**